app/Contact.php

```diff
@@ -228,7 +254,7 @@ public function getLastActivityDate($timezone)
         $lastActivity = $this->activities->sortByDesc('date_it_happened')->first();
 
         return DateHelper::getShortDate(
-            DateHelper::createDateFromFormat($lastActivity->date_it_happened, $timezone),
+            Carbon::parse($lastActivity->date_it_happened, $timezone),
             'en'
```

Comment:
We probably need to return the locale of the user that owns this contact instead here.